package-scripts/datadog-agent/postinst

```diff
@@ -57,10 +57,11 @@ if which chkconfig >/dev/null 2>&1; then
   chkconfig --add datadog-agent
 fi
 
+MISSING_CONF_MSG="\033[34m\n* No /etc/dd-agent/datadog.conf file has been found.\033[0m"
 if which invoke-rc.d >/dev/null 2>&1; then
-    invoke-rc.d datadog-agent restart
+    invoke-rc.d datadog-agent restart || echo "$MISSING_CONF_MSG"
 else
-    /etc/init.d/datadog-agent restart
+    /etc/init.d/datadog-agent restart || echo "$MISSING_CONF_MSG"
```

Comment:
Also when using escaped chars in a string you shouldn't use `echo`, but rather `printf`(you could theoretically use `echo -e` but it's not POSIX-compliant...).